pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -20,11 +21,8 @@ class SystemPrompt:
     content: str
     """The content of the prompt."""
 
-    role: Literal['user'] = 'user'
-    """Message source identifier, this type is available on all messages as either 'user' or 'model'."""
-
-    message_kind: Literal['system-prompt'] = 'system-prompt'
-    """Message type identifier, this type is available on all messages as a discriminator."""
+    kind: Literal['system-prompt'] = 'system-prompt'
```

Comment:
Could retain the note about "available on all message parts as a discriminator", same for the others in this file